dct_backend/server.py

```diff
@@ -1,29 +1,48 @@
 from fastapi import FastAPI
+from fastapi_utils.tasks import repeat_every
 from dct import DCT
+from webscraper import scrapeFrame, createSoup
+import sys
+
+def scrape_data(dct: DCT):
+    soup = createSoup("https://emoryatlanta.cafebonappetit.com/cafe/dobbs-common-table/")
+    output = scrapeFrame(soup)
+    dct.clear_data()
+
+    if "pytest" in sys.modules:
+        print("Using mocked data..")
+        dct.save_food_item('Pancakes','Stem To Root','Breakfast','100',['carbs'])
+        dct.save_food_item('Bagel', '605 Kitchen', 'Lunch', '100', ['carbs'])
+        dct.save_food_item('Cereal', 'Spice', 'Dinner', '102', ['vegan', 'carbs'])
+        dct.save_food_item('Sandwhich', 'Luncheonnette','Breakfast','102',['category1'])
+        dct.save_food_item('Wings', 'Stem To Root','Lunch','102',['category1'])
+        dct.save_food_item('Grilled Chicken', '605 Kitchen','Dinner','102',['vegan', 'carbs'])
+        dct.save_food_item('Beyond Beef Burger', 'Spice', 'Breakfast', '100',['carbs'])
+        dct.save_food_item('Portobello Mushrooms', 'Luncheonnette','Lunch','102',['vegan', 'carbs'])
+        dct.save_food_item('Macaroni & Cheese', 'Stem To Root','Dinner','102',['vegan', 'carbs'])
+        dct.save_food_item('Veggie Fried Rice', '605 Kitchen', 'Breakfast', '100',['carbs'])
+        dct.save_food_item('Avacado Grilled Cheese', 'Spice', 'Lunch', '100',['carbs'])
+        dct.save_food_item('Mediterranean Pita', 'Luncheonnette', 'Dinner', '100',['carbs'])
+        dct.save_food_item('Pasta', 'Stem To Root', 'Breakfast','100',['carbs'])
+        dct.save_food_item('French Fries', '605 Kitchen', 'Lunch', '100',['carbs'])
+        dct.save_food_item('Ceasar Salad', 'Spice','Dinner','102',['vegan', 'carbs'])
+
+        dct.save_serve_time('Breakfast', '7am - 10am')
+        dct.save_serve_time('Lunch', '11am - 2pm')
+        dct.save_serve_time('Dinner', '2pm - 5pm')
+    else:
+        for entry in output:
+            dct.save_food_item(*entry)
+        print("Successfully scrapped new data")
```

Comment:
did you mean "scraped"? Or did you actually mean scrapped, as in we discarded the new data?

Reply:
I can't english